utils/httputil/client.go

```diff
@@ -230,10 +234,34 @@ func (c *Client) request(
 			continue
 		}
 
-		if status = r.GetStatus(); status == StatusTooManyRequests || status >= 500 {
+		status = r.GetStatus()
+
+		if status == StatusNirnTimeout || status >= 500 {
```

Comment:
In the case of a 5xx response, I think we should do our own exponential backoff to prevent spamming Discord.